src/rules/no-entire-object-hook-deps.ts

```
@@ -457,15 +468,21 @@ export const noEntireObjectHookDeps = createRule<[], MessageIds>({
                   if (elementIndex === depsArg.elements.length - 1) {
                     const prevElement = depsArg.elements[elementIndex - 1];
                     if (prevElement) {
-                      const range: [number, number] = [prevElement.range![1], (element as TSESTree.Node).range![1]];
+                      const range: [number, number] = [
+                        prevElement.range![1],
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)